src/ola_copybook.txt

```diff
@@ -0,0 +1,450 @@
+                  05 W-T-DT-CC PIC 99.
+                  05 W-T-DT PIC 9(4).
+                  05 FILLER16 PIC 99.
+              03 W-DTE-LST-UTL PIC 9(6).
+              03 W-DTE-LST-UTL-REDF REDEFINES W-DTE-LST-UTL.
+                  05 W-DTE-LST-UTL-CCYY PIC 9(4).
+                  05 W-DTE-LST-UTL-CCYY-REDF REDEFINES
+                     W-DTE-LST-UTL-CCYY.
+                      07 W-DTE-LST-UTL-CC PIC 99.
+                      07 W-DTE-LST-UTL-YY PIC 99.
+                  05 W-DTE-LST-UTL-MM PIC 99.
+              03 W-NEW-WB-EXC PIC X(01).
+              03 W-NEW-WB-CONNECT-DATE PIC 9(8).
+              03 W-NEW-WB-CONNECT-DATE-REDF REDEFINES
+                 W-NEW-WB-CONNECT-DATE.
+                  05 W-NEW-WB-CCYY PIC 9(4).
+                  05 W-NEW-WB-CCYY-REDF REDEFINES
+                     W-NEW-WB-CCYY.
+                      07 W-NEW-WB-CC PIC 99.
+                      07 W-NEW-WB-YR PIC 99.
+                  05 W-NEW-WB-MO PIC 99.
+                  05 W-NEW-WB-DA PIC 99.
+              03 W-14B2-TYPE-COVERAGE PIC X(01).
+              03 W-14B2-APP-NO PIC 9(06).
+              03 FILLER17 PIC 9(04).
+              03 FILLER18 PIC 9(18).
+              03 FILLER19 PIC 9(07).
+              03 W-MONTH OCCURS 14 TIMES.
+                  05 W-MONTH-DATE PIC 9(6).
+                  05 W-MONTH-DATE-REDF REDEFINES W-MONTH-DATE.
+                      07 WM-CCYY PIC 9(4).
+                      07 WM-CCYY-REDF REDEFINES WM-CCYY.
+                          09 WM-CC PIC 99.
+                          09 WM-YR PIC 99.
+                      07 WM-MO PIC 99.
+                  05 WM-CHG PIC 9.
+                  05 WM-NO PIC 9.
+                  05 WM-ALLOW PIC S9(5) COMP-3.
+                  05 WM-ACODE PIC X.
+                  05 WM-TCODE PIC X.
+                  05 WM-LIMIT PIC S9(7) COMP-3.
+                  05 WM-ALLOW2 PIC S9(5) COMP-3.
+                  05 WM-ACODE2 PIC X.
+                  05 WM-TCODE2 PIC X.
+                  05 WM-LIMIT2 PIC S9(7) COMP-3.
+                  05 WM-DATE2 PIC 99.
+                  05 WM-ALLOW3 PIC S9(5) COMP-3.
+                  05 WM-ACODE3 PIC X.
+                  05 WM-TCODE3 PIC X.
+                  05 WM-LIMIT3 PIC S9(7) COMP-3.
+                  05 WM-DATE3 PIC 99.
+                  05 WM-FORM-LCK PIC 9.
+                  05 WM-PGT PIC S999 COMP-3.
+                  05 WM-TSWA PIC 9.
+                  05 WM-EGT PIC S999 COMP-3.
+                  05 WM-ESWA PIC 9.
+                  05 WM-ACRE PIC S999V99 COMP-3.
+                  05 WM-POTE PIC S9999V9 COMP-3.
+                  05 WM-ACFT PIC S9(5) COMP-3.
+                  05 WM-GOR PIC S9(5) COMP-3.
+                  05 WM-OTRAN-CD PIC 9.
+                  05 WM-POT PIC S999 COMP-3.
+                  05 WM-EOT PIC S999 COMP-3.
+                  05 WM-JOHN PIC 9.
+                  05 WM-OOIP PIC 9(06).
+                  05 FILLER20 PIC 9(03).
+
+WORKING-STORAGE SECTION.
```

Comment:
this program is a fully specified COBOL program.  it has to be in order to be parsed w/o syntax error by the generated Cobol85 parser.

i.e. if WORKING-STORAGE and PROCEDURE DIVISION. are not supplied, it's not a full COBOL program.
Could have changed the lexer to optionally allow these sections but did not want to.